packages/webpack-scaffold/README.md

```diff
@@ -2,7 +2,7 @@
 
 This is the utility suite for creating a webpack `scaffold`. It contains utility functions to assist with inquirer prompting and scaffolding.
 # Installation
-
+<!-- TODO: To be added as part of issue #712  -->
```

Comment:
Since we already have an issue dedicated for setting up installation guide, we don't need to add extra noise here.

Reply:
I add this to guide new contributors. So that they can easily find where to add installation guide.

Reply:
Should I remove it then @dhruvdutt